diff --git a/src/totem-object.c b/src/totem-object.c
index 2cc2fe018..fd71bb19e 100644
--- a/src/totem-object.c
+++ b/src/totem-object.c
@@ -67,11 +67,11 @@
 
 #define REWIND_OR_PREVIOUS 4000
 
-#define SEEK_FORWARD_SHORT_OFFSET 15
+#define SEEK_FORWARD_SHORT_OFFSET 5
 #define SEEK_BACKWARD_SHORT_OFFSET -5
 
-#define SEEK_FORWARD_LONG_OFFSET 10*60
-#define SEEK_BACKWARD_LONG_OFFSET -3*60
+#define SEEK_FORWARD_LONG_OFFSET 60
+#define SEEK_BACKWARD_LONG_OFFSET -60
 
 #define DEFAULT_WINDOW_W 650
 #define DEFAULT_WINDOW_H 500
diff --git a/src/totem-private.h b/src/totem-private.h
index 8e228b166..9d82a90c3 100644
--- a/src/totem-private.h
+++ b/src/totem-private.h
@@ -155,7 +155,7 @@ struct _TotemObject {
 
 GtkWidget *totem_volume_create (void);
 
-#define SEEK_FORWARD_OFFSET 60
+#define SEEK_FORWARD_OFFSET 15
 #define SEEK_BACKWARD_OFFSET -15
 
 #define VOLUME_DOWN_OFFSET (-0.08)
